frontend/packages/kubevirt-plugin/integration-tests/tests/models/rhvImportWizard.ts

```diff
@@ -110,27 +102,25 @@ export class RhvImportWizard extends ImportWizard {
       await this.fillDescription(description);
     }
     await this.next();
-    // Binding networks
-    // First update imported network interfaces to comply with k8s
-    await this.updateImportedNICs();
-    // Adding networks if any
-    if (networkResources) {
-      await this.addVmNetworks(networkResources);
-    }
-    await this.next();
+  }
 
-    // Binding storage disks
-    // First update disks that come from the source VM
-    await this.updateImportedDisks();
-    // Adding disks if any
-    if (storageResources) {
-      await this.addVmStorage(storageResources);
-    }
-    await this.next();
-    // CloudInit page is in read-only mode
-    await this.next();
-    // Additional devices page is in read-only mode
-    await this.next();
+  async import(config: VMImportConfig) {
+    const {
+      provider,
+      instanceConfig,
+      name,
+      description,
+      sourceVMName,
+      storageResources,
+      networkResources,
+      startOnCreation,
+    } = config;
+    await this.openWizard(VirtualMachineModel);
+
+    await this.importInitStep(provider, instanceConfig, sourceVMName);
+    await this.importConfigStep(name, description);
+    await this.importNetworkStep(networkResources);
+    await this.importDiskStep(storageResources);
 
     // Review
     await this.validateReviewTab(config);
```

Comment:
it would be nice if we could unify the config type with `VMBuilderData` or make looser type definitions in Wizard methods.

then we can easily use `processReviewStep` and `processAdvanceStep` (vmware). We can leave `processGeneralStep` for future refactoring.

Reply:
Please elaborate

Reply:
we don't have to be concerned about the types as they are optional now, although it would be nice to have it consistent later. But at least we could use  processReviewStep and processAdvanceStep here and in vmwareWizard

Reply:
I have refactored all those steps to use `config` as single parameter for each of them + I used `processAdvanceStep` and `processReviewStep` from wizard class to make my code more generic